server/build_event_protocol/build_event_server/build_event_server.go

```diff
@@ -184,7 +181,12 @@ func (s *BuildEventProtocolServer) PublishBuildToolEventStream(stream pepb.Publi
 }
 
 func postProcessStream(ctx context.Context, channel interfaces.BuildEventChannel, streamID *bepb.StreamId, acks []int, stream pepb.PublishBuildEvent_PublishBuildToolEventStreamServer) error {
-	if channel != nil && channel.GetNumDroppedEvents() > 0 {
+	if channel == nil {
+		log.CtxInfo(ctx, "Closing empty channel.")
```

Comment:
nit: maybe move this check outside of `postProcessStream` and change the info message to "No events received on stream"

Reply:
Done!